eth2util/signing/signing.go

```diff
@@ -229,3 +247,27 @@ func epochFromSlot(ctx context.Context, eth2Cl eth2wrap.Client, slot eth2p0.Slot
 
 	return eth2p0.Epoch(uint64(slot) / slotsPerEpoch), nil
 }
+
+// SSZUint64 implements ssz.HashRoot for uint64 types.
+type SSZUint64 uint64
```

Comment:
rather follow previous example of `eth2util.EpochHashRoot`

Reply:
sure